spec/dummy/db/schema.rb

```diff
@@ -10,10 +10,10 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2015_08_13_132804) do
-
+ActiveRecord::Schema[7.0].define(version: 2015_08_13_132804) do
```

Comment:
Is `Schema[7.0]` supported by the older Rails versions? The gem should be compatible with ActiveRecord 4+.